src/about-this-guide.md

```diff
@@ -4,26 +4,26 @@ This guide is meant to help document how rustc – the Rust compiler –
 works, as well as to help new contributors get involved in rustc
 development.
 
-There are six parts to this guide:
+There are seven parts to this guide:
 
 1. [Building and debugging `rustc`][p1]: Contains information that should be
    useful no matter how you are contributing, about building, debugging,
    profiling, etc.
-2. [Contributing to `rustc`][p1-5]: Contains information that should be useful
+2. [Contributing to `rustc`][contributing]: Contains information that should be useful
```

Comment:
nit: I'd like to keep the consistency of the links here. I mean, the chapter name vs. the chapter number. I'm fine either way.

Reply:
Actually I'm not quite sure which chapter p1-5 is referring to? I'm not quite sure what  chapter number to use in this place.

Reply:
I guess it was considered as a subchapter? It explains why it said "six chapters". But now it has its own chapter so it should make sense to call it p2.

Reply:
Fixed.